### PR TITLE
librasterlite2: fix clang build

### PR DESCRIPTION
**Pull request by raedrizqie**

### CODE REVIEW

**Comment by Biswa96** [comment]
Is it required to add Makefile.in diff in patch file? AFAIK, Makefile.in is automatically generated from Makefile.am while running autoreconf command.

**Comment by raedrizqie** [comment]
> Is it required to add Makefile.in diff in patch file? AFAIK, Makefile.in is automatically generated from Makefile.am while running autoreconf command.

yes, you are right.. i will update it later

File: mingw-w64-librasterlite2/001-makefile-fix.patch [hunk 1]
```diff
@@ -0,0 +1,56 @@
+diff -urN librasterlite2-1.1.0-beta1-orig/src/Makefile.am librasterlite2-1.1.0-beta1/src/Makefile.am
+--- librasterlite2-1.1.0-beta1-orig/src/Makefile.am	2020-08-24 23:20:40 +0800
++++ librasterlite2-1.1.0-beta1/src/Makefile.am	2021-11-08 13:18:05 +0800
+@@ -26,13 +26,8 @@
+ 	@LIBFREETYPE2_LIBS@ @LIBPIXMAN_LIBS@ @LIBCAIRO_LIBS@ \
+ 	@LIBSPATIALITE_LIBS@
+ 
+-if MINGW
```
Comment (lazka) [inline mingw-w64-librasterlite2/001-makefile-fix.patch:8]: Are those MINGW checks not working? I'm wondering why you removed them

Comment (raedrizqie) [reply mingw-w64-librasterlite2/001-makefile-fix.patch:8]: not working for clang (at least for me).. maybe you could help me out :)

Comment (raedrizqie) [reply mingw-w64-librasterlite2/001-makefile-fix.patch]: patch is updated

Comment (lazka) [reply mingw-w64-librasterlite2/001-makefile-fix.patch]: nice

**Comment by lazka** [comment]
clang doesn't build

**Pull request merged**